From 2bd298dec79bd30aa0d54a5ca1d519cb9a36a230 Mon Sep 17 00:00:00 2001
From: Jan Gaspar <jano_gaspar@yahoo.com>
Date: Mon, 26 Apr 2010 09:40:08 +0000
Subject: [PATCH] circular_buffer: applied patch to remove workaround for DEC
 C++ compiler - now all compilers will use 
 this->invalidate_iterators_except

[SVN r61570]
---
 include/boost/circular_buffer/space_optimized.hpp |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/include/boost/circular_buffer/space_optimized.hpp b/include/boost/circular_buffer/space_optimized.hpp
index 33008e8..9c76afe 100644
--- a/include/boost/circular_buffer/space_optimized.hpp
+++ b/include/boost/circular_buffer/space_optimized.hpp
@@ -1260,11 +1260,7 @@ private:
                 ensure_reserve(new_capacity, new_size));
         }
 #if BOOST_CB_ENABLE_DEBUG
-# if BOOST_WORKAROUND(__DECCXX_VER, BOOST_TESTED_AT(70190006))
         this->invalidate_iterators_except(end());
-# else
-        invalidate_iterators_except(end());
-# endif
 #endif
     }
 
@@ -1284,11 +1280,7 @@ private:
         circular_buffer<T, Alloc>::set_capacity(
             ensure_reserve(new_capacity, size()));
 #if BOOST_CB_ENABLE_DEBUG
-# if BOOST_WORKAROUND(__DECCXX_VER, BOOST_TESTED_AT(70190006))
         this->invalidate_iterators_except(end());
-# else
-        invalidate_iterators_except(end());
-# endif
 #endif
     }
 
-- 
1.7.6.5

